.github/workflows/rit.yml

```diff
@@ -94,6 +94,14 @@ jobs:
             exit 1
           fi
 
+          # Set the Repo Owner
+          if [ -n "${{ github.event.pull_request.head.repo.owner.login }}" ]; then
+            REPO_OWNER=${{ github.event.pull_request.head.repo.owner.login }}
+          else
+            REPO_OWNER=${{ github.repository_owner }}
+          fi
```

Comment:
suggestion:

Instead of have an if-else, we could set the default `REPO_OWNER` in the line 45 of this file and then we could override the value here if the owner variable is defined. Just to follow the approach used in the whole script. Also, would be good to define these github variables inputs like the other ones, just to have a single approach for the whole pipeline.

With these recommendations we would need the following changes:

1 - Line 36 to add both github variables:

```
          github_event_pull_request_head_repo_owner_login : ${{ github.event.pull_request.head.repo.owner.login }}
          github_repository_owner: ${{ github.repository_owner }}

``` 

2 - Line 45 set the default repo owner:

```
          default_repo_owner="$github_repository_owner"
```

3 - Line 97, change this if else for the following:

```
          # Set the Repo Owner
          if [ -n "$github_event_pull_request_head_repo_owner_login" ]; then
            REPO_OWNER="$github_event_pull_request_head_repo_owner_login"
          fi
```

Reply:
Thank you for the feedback! I’ve made the following changes based on your suggestions:

1. Defined `github_event_pull_request_head_repo_owner_login` and `github_repository_owner` as environment variables at the beginning for consistency.
2. Set `REPO_OWNER` using the one-liner: `REPO_OWNER="${github_event_pull_request_head_repo_owner_login:-$github_repository_owner}"`, which simplifies the logic by defaulting to github_repository_owner if the pull request owner isn’t available.

Let me know what you think of these changes, or if you have any other suggestions!